README.md

```diff
@@ -1,4 +1,14 @@
 # Smart-Mirror
 Raspberry powered mirror which can display the news, weather, and time.
 
+## Installation and Updating
+Installation is simple as running the following command.
+
+> sudo pip install git+https://github.com/HackerHouseYT/Smart-Mirror.git
+
+## Running
+To run the application run the following command in this folder
+> python smartmirror.py
```

Comment:
I think `pyhton smartmirror.py --token TOKEN_KEY` would be tedious if you have to append the token every time you run it. Let's keep it as is for now.